simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp

```diff
@@ -41,9 +40,9 @@ autoware_auto_vehicle_msgs::msg::VelocityReport to_velocity_report(
   const std::shared_ptr<SimModelInterface> vehicle_model_ptr)
 {
   autoware_auto_vehicle_msgs::msg::VelocityReport velocity;
-  velocity.longitudinal_velocity = static_cast<float32_t>(vehicle_model_ptr->getVx());
+  velocity.longitudinal_velocity = static_cast<double>(vehicle_model_ptr->getVx());
   velocity.lateral_velocity = 0.0F;
-  velocity.heading_rate = static_cast<float32_t>(vehicle_model_ptr->getWz());
+  velocity.heading_rate = static_cast<double>(vehicle_model_ptr->getWz());
   return velocity;
```

Comment:
It is because the module interface message was defined by float_32 (e.g. path velocity). This package implementation just followed the interface. (If I remember correctly)

Reply:
I don't know much about the risk of using different types, but if it is working, I think it is fine. There is no need to revert your commit. I just wanted to make sure it wasn't an unintentional change.

Reply:
:+1: